tests/functional/adapter/test_aliases.py

```diff
@@ -0,0 +1,24 @@
+import pytest
+from dbt.tests.adapter.aliases.test_aliases import TestAliases, TestSameAliasDifferentSchemas
```

Comment:
We should Import the base class here not the Test class I think?

Reply:
that's it thank you, slipped my mind will swap to base classes in core